client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -213,6 +213,12 @@ class AdsFormSettings extends Component {
 		}, 0 );
 	}
 
+	jetpckPlacementControls() {
```

Comment:
Typo? :)

```suggestion
	jetpackPlacementControls() {
```

(same typo with line 543 though)

Reply:
good catch ;)